3p/iframe-transport-client.js

```diff
@@ -83,7 +83,20 @@ export class IframeTransportClient {
   }
 
   /**
-   * Gets the IframeMessagingClient
+   * Sends a message back to the creative.
+   * @param {string} vendor The name of the 3p vendor used in the
+   *   <amp-analytics> tag
+   * @param {string} creativeId An ID uniquely identifying which creative
+   * shall receive the event
+   * @param {!Object<string,string>} response
+   */
+  sendMessageToCreative(vendor, creativeId, response) {
```

Comment:
I think `vendor` is fixed when creating the `iframeTransportClient`, can they specify `vendor` here?

Reply:
Keith and I favor wrapping vendor and creativeId in a closure, so that they only specify response. I attempted to do that in an earlier iteration, but @lannka felt it was too complex. I'll change it to try to encapsulate, but more simply than before.

Reply:
@zhouyx I spoke to Keith about this, and we'd like to change the interface to more closely resemble an earlier version. The remote frame side would look very similar to this: https://github.com/ampproject/amphtml/pull/10184/files#diff-ea997293239e1df81343e40e465c4dcdR17 (except that events would still be passed one-by-one, so no need for the for loop).
The library side from that old PR is at https://github.com/ampproject/amphtml/pull/10184/files#diff-65167c4b016e67bd1ccdbbb1a3a0e99fR33 . This is the one that Hongfei felt was too complex, so I will work on greatly simplifying it, but retaining the general closure concept.
Another group is depending on this with a pretty tight schedule, so I'll begin work on this but please let me know if you have any objections/concerns. Thanks!

Reply:
Finished my part quickly, so went ahead and committed. Please let me know your thoughts :)